setup.py

```diff
@@ -158,21 +159,27 @@ def __init__(self, compiler=None):
 
     def get_shared_lib_extension(self) -> str:
         """Returns shared library file extension"""
-        if sys.platform.startswith('win'):
+        if sys.platform == 'win32':
             return '.dll'
-        elif sys.platform.startswith('linux'):
+        elif sys.platform == 'linux':
             return '.so'
-        elif sys.platform.startswith('darwin'):
+        elif sys.platform == 'darwin':
             return '.dylib'
         else:  # Return same value as used by build_ext.get_ext_filename
             return sysconfig.get_config_var('EXT_SUFFIX')
 
     def has_cpp11(self) -> bool:
         """Check C++11 availability on host"""
         if self.__compiler.compiler_type == 'msvc':
-            return sys.version_info[:2] >= (3, 5)
+            return True
         return check_compile_flags(self.__compiler, '-std=c++11', extension='.cc')
 
+    def has_cpp14(self) -> bool:
+        """Check C++14 availability on host"""
+        if self.__compiler.compiler_type == 'msvc':
+            return True  # TODO check if correct
```

Comment:
Is this true?

Reply:
It works without any problem with the versions recommended by python org.

MS says "enjoy C++11, C++14 and features of C++17"